includes/class-wc-gateway-stripe.php

```diff
@@ -336,10 +347,48 @@ public function elements_form() {
 		<?php
 	}
 
+	/**
+	 * Returns the URL of the configuration screen for this gateway, for use in internal links.
+	 *
+	 * @return string URL of the configuration screen for this gateway
+	 */
+	public static function get_settings_url() {
+		return admin_url( add_query_arg( self::$settings_url_params, 'admin.php' ) );
+	}
+
+	/**
+	 * Output the React Settings page wrapper
+	 */
+	private function output_settings_page_wrapper() {
+		global $hide_save_button;
+		$hide_save_button = true;
+		?>
+		<div>
+			<ul class="subsubsub">
+				<li><a href="<?php echo esc_html( self::get_settings_url() ); ?>" class="current"><?php echo esc_html( $this->get_method_title() ); ?></a></li>
+				<li>|</li>
+				<li><a href="<?php echo esc_html( admin_url( 'admin.php?page=wc-settings&tab=checkout' ) ); ?>"><?php echo esc_html( __( 'All payment methods', 'woocommerce-gateway-stripe' ) ); ?></a></li>
+			</ul>
+		</div>
+		<div id="wc-stripe-account-settings-container"></div>
+		<?php
+	}
+
+	/**
+	 * Maybe override the parent admin_options method.
+	 */
+	public function admin_options() {
+		if ( ! WC_Stripe_Features::is_upe_enabled() ) {
```

Comment:
Looks like this branch needs an update from `develop`. If this branch gets merged, a fatal error will occur because the `WC_Stripe_Features` no longer exists, it has been renamed to `WC_Stripe_Feature_Flags`